src/IconButton/index.jsx

```diff
@@ -11,32 +11,32 @@ const IconButton = ({
   src,
   iconClassNames,
   onClick,
+  size,
   variant,
   iconAs: IconComponent,
   ...attrs
 }) => {
-  const invertedHoverClass = invertColors ? `iconbutton-hover__${variant}--invert` : '';
-  const invertedIconClass = invertColors ? `iconbutton__${variant}--invert` : '';
-
+  const invert = invertColors ? 'inverse-' : '';
   return (
     <button
       {...attrs}
       aria-label={alt}
       className={classNames(
-        'iconbutton-hover',
-        invertedHoverClass,
-        `iconbutton-hover__${variant}`,
+        'btn-icon',
+        `btn-icon-${invert}${variant}`,
+        `btn-icon-${size}`,
         attrs.className,
       )}
       onClick={onClick}
       type="button"
     >
-      <IconComponent
-        className={`iconbutton iconbutton__${variant} ${invertedIconClass} ${iconClassNames}`}
-        icon={icon}
-        alt={alt}
```

Comment:
Chose to remove the `alt` because the icon in this scenario is purely decorative, and the aria-label on the button should describe the button's function.